pages/_app.tsx

```diff
@@ -80,9 +85,47 @@ const InitializePrice = React.memo(function InitializePrice() {
   return <></>
 })
 
-function MyApp({ Component, pageProps, router }: AppProps) {
-  console.log('here', { ...pageProps })
+const InitializeAuth = () => {
```

Comment:
ah so this removes the hard block (where you can't even see the pages on portal) and makes it more like opyn.co, makes sense, otherwise strikes aren't possible/useful

Reply:
yesss